scripts/create_addon

```diff
@@ -101,10 +101,16 @@ pack_addon() {
   if [ ! -z "$PKG_MAINTAINER" ] ; then
     PROVIDER_NAME="$PKG_MAINTAINER"
   fi
+
   ADDON_NAME="$PKG_NAME"
   if [ ! -z "$PKG_ADDON_NAME" ] ; then
     ADDON_NAME="$PKG_ADDON_NAME"
   fi
+
+  if [ ! -z "$PKG_ADDON_BROKEN" ] ; then
+    PKG_BROKEN="$PKG_ADDON_BROKEN"
+  fi
```

Comment:
PKG_BROKEN is redundant - it's not used anywhere.

Reply:
used here https://github.com/LibreELEC/LibreELEC.tv/pull/2067/files#diff-f9aa64ed568e7dbfced28114c6fe1df0R23

Reply:
You're using `$PKG_ADDON_BROKEN` in the `sed` below, not `$PKG_BROKEN`, so `$PKG_BROKEN` is being set but never used. And it's not required, the `sed` below is fine as it is - the 4 lines above can be dropped.